setup.py

```diff
@@ -3,37 +3,45 @@
 import io
 import sys
 from setuptools import setup, Extension
+from Cython.Build import cythonize
```

Comment:
If Cython is imported in `setup.py`, the users of `sdist` packages need to install Cython before installing this package. Therefore, I think it might be better to avoid importing Cython here by running the `cython --cplus mojimoji.pyx` command (previously contained in the `cythonize.sh`) in the workflow.

Reply:
The output of `cythonize` is platform-dependent, which is a cause of the previous mojimoji's intallatation problem.
Instead of restoring `cythonize.sh` and `mojimoji.cpp`, I added `pyproject.toml` to the root directory so that users can install mojimoji without cython. 

Please see here for more information on how it works.
https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#basic-setup-py

Reply:
Thank you for your reply and your prompt work to deal with the reviews!